dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralIT.java

```diff
@@ -926,7 +926,7 @@ public void testToolAlias() {
         DockstoreTool refresh = containersApi.refresh(tool.getId());
 
         // Add alias
-        Entry entry = entryApi.updateAliases(refresh.getId(), "foobar", "");
+        Entry entry = entryApi.addAliases(refresh.getId(), "foobar");
```

Comment:
Correct me if I am wrong but PUT requires an empty body parameter to appease swagger. Since we now use POST the body parameter is no longer needed; e.g.
io/swagger/client/api/OrganizationsApi.java:300
   * @param body This is here to appease Swagger. It requires PUT methods to have a body, even if it is empty. Please leave it empty. (optional)

This was the change in Entry.java:
-    default T updateAliases(User user, Long id, String aliases, String emptyBody) {
+    default T addAliases(User user, Long id, String aliases) {

Reply:
Ah, I see.